app/controllers/shipments_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ShipmentsController < ApplicationController
   def index
     company = Company.find(params[:company_id])
-    @shipments = Shipment.includes(:shipment_items).where(company_id: company.id)
+    @shipments = paginate(Shipment.includes(:shipment_items).where(company_id: company.id), per_page: 2)
```

Comment:
```suggestion
    @shipments = paginate(company.shipments.includes(:shipment_items), per_page: 2)
```
Also, do you have a reason that a page only has 2 items?

Reply:
It's good to utilize the tool like `paginate` here.
But could you explain a bit more how it works? For example, usually, a pagination function should work with `per_page` and `off_set` then it can return not only items but also from which item.

Reply:
Yes, I'll add a comment on controller to explain how to use `page` and `per_page` params to return required information, thanks a lot for reminding me!